### ISSUE TITLE
Defaulting to Low VRAM on Nvidia 3080 TI

### ISSUE THREAD

**Issue by ritmototal** [comfyanonymous/ComfyUI_examples#30]
Hello both with Stabel Diffusion 3 and Flux I keep getting pushed into low vram mode with just about anything I try to generate.

I am quite new to this, and maybe there is something wrong with my ComfyUI installation, but I could also be doing something wrong. I am running the software with the Nvidia GPU option, and below are my hardware specs:
Device name	JRS2474S3-DG-76
Processor	AMD Ryzen 7 7700X 8-Core Processor      4.50 GHz
Installed RAM	32.0 GB (31.1 GB usable) DDR6 6000 MHZ

If anyone has any information on this or suggestions please let me know.

**Comment by comfyanonymous** [comment]
That should be fixed now, try updating, (update/update_comfyui.bat) on the standalone.

**Comment by ritmototal** [comment]
I have run the update bat but looks like the issue persists 
Loading 1 new model
loading in lowvram mode 9771.075

**Comment by comfyanonymous** [comment]
oh that's normal then, flux doesn't fit on 12GB vram.

**Comment by ritmototal** [comment]
I am using the smaller one but mind you this problem was also happening on Stable Diffusion 3 with just about any setting.

I have found another error that could be related to this from reading online could be slowing it down significantly as well but not sure how to fix it with what I have read so far:

C:\Program Files\ComfyUI\ComfyUI\comfy\ldm\modules\attention.py:407: UserWarning: 1Torch was not compiled with flash attention. (Triggered internally at ..\aten\src\ATen\native\transformers\cuda\sdp_utils.cpp:455.)
  out = torch.nn.functional.scaled_dot_product_attention(q, k, v, attn_mask=mask, dropout_p=0.0, is_causal=False)
